docs/mission_module.md

```diff
@@ -0,0 +1,118 @@
+# Supported devices
+
+The ONLY supported device is the Autonomy Device, serving as a driver for the car. The device is connected to the Mission Module and receives the mission to drive the car to the desired stops.
+
+This module handles backward compatibility with Fleet protocol v1.2.0. This version did not have modules and supported only one type of payload - CarStateProtocol, which contains statuses from autonomy and commands for autonomy. The payload for the module is serialized by protobuf v 3.21.12, the same version as the fleet protocol itself.
+
+Device ID currently has the following field values:
+- module ID: 1,
```

Comment:
It would be nice to have these informations also as part of the mian readme I do not know how to use module, what is module id etc.

When I open a Module I need to have informatios as stated in the issue I have created.

When I came to to git repo I need to knownthe meaning, how the project fits into the system etc. IN main readme there are almist no informations which tells me WHY, HOW, WHAT.